service/src/main/java/uk/gov/hmcts/reform/rd/client/OrganisationApi.java

```diff
@@ -39,4 +39,11 @@ Organisation findUserOrganisation(
         @RequestHeader(AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization
     );
+
+    @GetMapping("/refdata/internal/v1/organisations")
+    Organisation findManagedUserOrganisation(
```

Comment:
i would name it findOrganisation, it is not about  current user organisation, it is about finding org by id